contributing/code/bc.rst

```diff
@@ -334,84 +334,145 @@ Changing Traits
-Remove protected property                           No
-Reduce visibility                                   No
-Make public                                         No
-Move to a used trait                                Yes
+Add protected property                                                           Yes
+Remove protected property                                                        No
+Reduce visibility                                                                No
+Make public                                                                      No
+Move to a used trait                                                             Yes
 **Private Properties**
-Add private property                                Yes
-Remove private property                             No
-Make public or protected                            Yes
-Move to a used trait                                Yes
+Add private property                                                             Yes
+Remove private property                                                          No
+Make public or protected                                                         Yes
+Move to a used trait                                                             Yes
 **Constructors and destructors**
-Have constructor or destructor                      No
+Have constructor or destructor                                                   No
 **Public Methods**
-Add public method                                   Yes
-Remove public method                                No
-Change name                                         No
-Reduce visibility                                   No
-Make final                                          No              :ref:`[6] <note-6>`
-Move to used trait                                  Yes
-Add argument without a default value                No
-Add argument with a default value                   No
-Remove argument                                     No
-Add default value to an argument                    No
-Remove default value of an argument                 No
-Add type hint to an argument                        No
-Remove type hint of an argument                     No
-Change argument type                                No
-Change return type                                  No
+Add public method                                                                Yes
+Remove public method                                                             No
+Change name                                                                      No
+Reduce visibility                                                                No
+Make final                                                                       No              :ref:`[6] <note-6>`
+Move to used trait                                                               Yes
+:ref:`Add argument with a default value <add-argument-public-method>`            No
```

Comment:
changing `without` to `with` looks wrong to me

Reply:
oops